README.md

```diff
@@ -143,6 +143,43 @@ following scripts are provided:
   found, "openqa-investigate". Compare to section
   "Combine auto-review and openqa-investigate"
 
+### openqa-investigate comments
+
+Let's say we have a failed test `openqa-Tumbleweed-dev-x86_64-Build:TW.315-vim`.
+openqa-investigate will trigger several jobs and write a comment:
+
+```
+Automatic investigation jobs for job 123:
+* vim:investigate:retry: https://openqa/t124
+* vim:investigate:last_good_tests:abc123: https://openqa/t126
+* vim:investigate:last_good_build:TW.314: https://openqa/t125
+* vim:investigate:last_good_tests_and_build:abc123+TW.314: https://openqa/t127
+```
+
+The first job `vim:investigate:retry` is a simple retry with the same settings,
+in order to find out if the failure is reproducible consistently.
+
+The `vim:investigate:last_good_tests:abc123` is a retry which uses the last
+good test code. It searches for the last good test, and if there were changes
+in the test code, it will trigger the test using the git sha from that last good
+test, e.g. `CASEDIR=orginal_git_url#abc123`.
+
+The `vim:investigate:last_good_build:TW.314` is a retry using the last good
+build.  It searches for the last good test, and if it has a different build
```

Comment:
```suggestion
build. It searches for the last good test, and if it has a different build
```

Reply:
I'm using the vim `gq` mapping to wrap blocks, it inserts those spaces. They will be collapsed to one space anyway. Do I need to manually delete them?
Maybe I'm using `gq` wrongly?

Reply:
Well, I have the same experience and I am deleting them manually by searching for double spaces if they don't jump to my eye already in before :)